stl/inc/chrono

```diff
@@ -5480,8 +5480,11 @@ namespace chrono {
         if constexpr (is_same_v<_Clock, utc_clock>) {
             if (_CHRONO get_leap_second_info(_Val).is_leap_second) {
                 _Os << _STATICALLY_WIDEN(_CharT, "60");
-                return;
+            } else {
+                const auto _Dp = _CHRONO floor<days>(_Val) + get_leap_second_info(_Val).elapsed;

```

Comment:
This should be `_CHRONO get_leap_second_info(_Val)` - but there's a call immediately above on line 5481, so it should be extracted instead of called twice.